packages/webapp/src/containers/Finances/styles.module.scss

```diff
@@ -29,6 +29,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  overflow: auto;
```

Comment:
I wasn't sure about this but I think ultimately we'd want to keep all cards the same size so that switching doesn't look weird, and we'll need the cards to be a certain width for the contents to fit nicely -- so in the end I thought it might be better to keep them a fixed width and make the contents scrollable for smaller mobile screens. Wdyt?

Reply:
The animation looks better, but personally, I think it looks nicer without scrollbar. But it doesn't make sense if the content doesn't show all the information...
I would ask Loïc for suggestions, but I guess making it scrollable is the best that could be done at the moment, so I'll approve!